content/en/docs/Installation/_index.md

```diff
@@ -40,6 +40,8 @@ helm repo update
 
 Use Helm 3.2+ to create a Namespace and install Kyverno.
 
+Use Kyverno 1.4.0+ to create customizable Helm release name while installing Kyverno. Before Kyverno 1.4.0, Helm release name must be Kyverno.
```

Comment:
Needs articles:

* "...to create a customizable"
* "...the Helm release name..."